src/graphql/queries.js

```diff
@@ -111,6 +111,11 @@ fragment WorkflowData on Workflow {
   port
 }
 
+fragment CyclePointData on FamilyProxy {
+  id
+  cyclePoint
+}
```

Comment:
A new fragment for cycle points. Notice that it is built from a `FamilyProxy`, so effectively the `id` is a `FamilyProxy` ID